LKMEX-whitelist/ElrondBuddies.md

```diff
@@ -0,0 +1,48 @@
+## Name of the project
+Elrond Buddies
+
+## Description of the project
+Elrond Buddies is a NFT project composed of 10,000 unique crypto collectibles on MultiversX.
+
+website: https://www.elrondbuddies.com/
+
+**Old PR**: https://github.com/ElrondNetwork/elrond-specs/pull/19
+
+## Purpose
+We send to our holders monthly rewards with LKMEX, UTK, RIDE, ITHEUM and WATER.
+
+To farm LKMEX on the dex we use this following wallet called 'ebuddiesdao', every month 20% of what we generated is reinvested in the LKMEX farm and we distribute 80% to the holders.
+https://explorer.elrond.com/accounts/erd16n5yll0yver02j4wm0l5fzw79c0dqg8ra3exq3ag35n7tfuwwxksxxzwzt
+
+We started distributing LKMEX on the 21/04/2022, so far we have distributed 73,002,098 LKMEX to our holders and reinvested 14,756,856 LKMEX
+
+We used two python scripts that we developed to get the holders and to distribute them the LKMEX:
+
+https://github.com/xdevguild/multiversX-nft-holders
+https://github.com/xdevguild/esdt-and-lkmek-airdrop-scripts
+
+Now that you guys want us to use a SC to distribute LKMEX we found a new way to do it with it.
+
+## Address of Smart Contract (Devnet)
+https://devnet-explorer.elrond.com/accounts/erd1qqqqqqqqqqqqqpgqwk5lgpf2fdf2veej683ltr2gzcyr0al5y7zswazjc5
```

Comment:
We cannot approve this address as this is on the devnet. We need on mainnet. Follow the guidelines.

Reply:
I don't get you ! You rejecting us, and here you tell me I just need to deploy it on mainnet to be good ?!